decisions/0002-record-architecture-decisions.md

```diff
@@ -0,0 +1,49 @@
+# 2. Communicating NIST's OSCAL Events to the Public
```

Comment:
B: The name of this file is misleading. It should be `0002-communicating-oscal-events.md` or the like.

Reply:
Thanks, I can rename it. Somehow I missed that.

Reply:
Renamed in https://github.com/usnistgov/OSCAL/pull/1638/commits/1f30aeabb6caec56203f3ebf7c80217b190643c2.